pkg/placement/helpers/clusters.go

```diff
@@ -63,7 +66,7 @@ func (c *ClusterSelector) Matches(ctx context.Context, cluster *clusterapiv1.Man
 
 	// match with cel selector if exists
 	if c.celSelector != nil {
-		if ok := c.celSelector.Validate(ctx, cluster); !ok {
+		if ok, _ := c.celSelector.Validate(ctx, cluster, celconfig.RuntimeCELCostBudget); !ok {
```

Comment:
why we need this as a input?

Reply:
This is for UT to input a small budget and test "running out of cost budget" case.

Reply:
but it seems like a global var that we can also set during ut?

Reply:
Yes! Modified this part to set CostBudget as a global var.